src/EnrollChoicesController.js

```diff
@@ -47,13 +47,27 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
         case Enums.ALL_OPTIONAL_SOME_ENROLLED:
         case Enums.HAS_REQUIRED_ALL_REQUIRED_ENROLLED:
           return Okta.loc('enroll.choices.optional', 'login');
+        case Enums.HAS_REQUIRED_SOME_REQUIRED_ENROLLED:
+          var remainingDays = this.options.appState.get('gracePeriodRemainingDays');
+          return (Number.isInteger(remainingDays) && remainingDays >= 0) ?
```

Comment:
I think `gracePeriodRemainingDays` should return an integer all the time, with value 0 or above. At the end, `0` means required right now... Unless we need to know that the value is not actually set, which is where returning `null` will help. But I don't think we need that

Reply:
talked offline, makes sense as is. `null` means no grace period, and then any number will be handled in `getGracePeriodSubtitle`, where `0` means we still are in grace period